src/controllers/user.js

```diff
@@ -48,10 +51,34 @@ const deleteUser = async (req, res) => {
   res.status(204).end()
 }
 
+const deactivateUser = async (req, res) => {
+  const { role, id: currentUserId } = req.user
+  const { id } = req.params
+
+  if (id !== currentUserId) throw createForbiddenError()
+
+  await userService.updateStatus(id, { [role]: STATUS_ENUM[2] })
```

Comment:
I suggest moving `STATUS_ENUM[2]` to separate constant for better readability.

Example:

```js
const DEACTIVATED_STATUS = STATUS_ENUM[2];
await userService.updateStatus(id, { [role]: DEACTIVATED_STATUS  })
```

Reply:
Thanks, now it looks better